src/bin/initdb/initdb.c

```diff
@@ -2665,7 +2666,8 @@ usage(const char *progname)
 	printf(_("\nLess commonly used options:\n"));
 	printf(_("  -d, --debug               generate lots of debugging output\n"));
 	printf(_("  -k, --data-checksums      use data page checksums\n"));
-	printf(_("  --shared-preload-libraries load libraries, same as shared_preload_libraries in postgresql.conf\n"));
+	printf(_("  --shared-preload-libraries load libraries, same as shared_preload_libraries in postgresql.conf.\n"
```

Comment:
Do we have good reasons to keep the `--shared-preload-libraries`?

Reply:
show what we modified in the help message

Reply:
Just remove `--shared-preload-libraries`. The env should be documented on our website, and a unique way for users to set it should be good enough (and less confusion).